backend/src/main/java/com/bakdata/conquery/models/query/queryplan/specific/ConceptNode.java

```diff
@@ -52,7 +54,20 @@ public void nextBlock(Bucket bucket) {
 			super.nextBlock(bucket);
 		}
 	}
-	
+
+	@Override
+	public boolean isOfInterest(Entity entity) {
+		for (ConceptElement<?> element : concepts) {
+			for (Connector connector : element.getConcept().getConnectors()) {
+				if(entity.has(connector.getId())) {
+					return true;
+				}
+			}
+		}
+
+		return false;
```

Comment:
```suggestion
		return entity.has(table.getResolvedConnector().getId()));
```

Reply:
Wie du in der for-Loop nach dem Connector suchst gibt die Funktion auch `true` zurück, wenn der QueryPlan/ConceptNode gar nicht auf die Tabelle verweißt , sondern eine andere Tabelle ausgewählt ist.